mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/ChessFacade.kt

```diff
@@ -36,7 +36,7 @@ class ChessFacade(private val auth: Auth, private val store: Store) {
   }
 
   /**
-   * Returns the [Match] associated to the given identifier.
+   * <<<<<<< HEAD Returns the [Match] associated to the given identifier.
```

Comment:
Looks like a VCS issue.

```suggestion
   * Returns the [Match] associated to the given identifier.
```